### PR TITLE
Added functions for tagging comment for escalation

### PR DESCRIPTION
**Pull request by jioGRAPHI**
### CHANGES
- added checkbox on comment area to mark current comment for escalation
- added a flag and is_elevated on CommentForm.vue
- added is_elevated field on the form being sent on submitComment() function on OpportunityInfo.vue
- added is_elevated in the post request URL for addComment() on the services\Project.vue
- add is_elevated on the addComment() in store\project\actions.js

### CODE REVIEW

File: src/components/comment/CommentForm.vue [hunk 1]
```diff
@@ -31,7 +39,9 @@ export default {
   },
   methods: {
     handleSubmit() {
-      this.$emit("onSubmit", this.comment);
+      this.flag? this.is_elevated = 1: this.is_elevated = 0;
+      this.$emit("onSubmit", this.comment, this.is_elevated);
+      this.flag? this.flag = !this.flag: this.flag = this.flag;
```
Comment (Gyakusetsu) [inline src/components/comment/CommentForm.vue]: change this to 
this.flag = !this.flag;

- and what is the difference of flag and is_elevated

Comment (jioGRAPHI) [reply src/components/comment/CommentForm.vue]: - magiging true yung value ng flag sa next comment kung magcocoment ng hindi elevated kapag ginawang `this.flag = !this.flag;`. 

- Kaya `this.flag? this.flag = !this.flag: this.flag = this.flag;` para nakadefault na false yung flag after ng every submit ng comment.

- `flag` yung magdedetermine if elevated yung comment or not.
- `is_elevated` yung ipapasa na value sa db (0 or 1) depende sa `flag`

**Comment by mhackyu** [comment]
Hi @jioGRAPHI ,

upon clicking of `Elevate Comment` checkbox this error comes up 
image


---

**comment list for elevated comment**
@hakunozen is there any icon or UI improvements that we need to add for elevated comments?
image

**Comment by jioGRAPHI** [comment]
Issue resolved

**Comment by mhackyu** [comment]
@hakunozen waiting for the UI update before merging this PR.

**Pull request merged**